tests/integration/test_eda.py

```diff
@@ -50,13 +50,13 @@ def test_eda_import(workdir, local_image_config):
 
     # the data should have all the relevant bits
     assert results["contents"] == [
-        {"content_type": "playbook", "description": None, "name": "hello.yml"},
-        {"content_type": "role", "description": "your role description", "name": "test_role"},
         {
             "content_type": "module",
             "description": "Upper cases a passed in string",
             "name": "upcase",
         },
+        {"content_type": "playbook", "description": None, "name": "hello.yml"},
+        {"content_type": "role", "description": "your role description", "name": "test_role"},
     ]
```

Comment:
Can a test be added with the specific documentation type?

Can we generate or mock a collection with eda extension stuff and assert on it?

IIUC this test change is just reordering right?

Reply:
nm, I saw #330 now

Reply:
This isn't covering EDA, but this is just reordering.

The test changes in `tests/integration/test_collections.py` have been updated to make sure we are picking up the documentation for filter plugins that were missed previously.

In my other PR, instead of re-ordering, I've made the test perform a sort for comparisons:

https://github.com/ansible/galaxy-importer/pull/330/files#diff-bd15eaea92516e3fc24670f037e19ce66e9cf60187603a827e123bf43f5d49a7L52

The reason this was required was due to a change I made at https://github.com/ansible/galaxy-importer/pull/329/files#diff-9d660c36abac0799a93b9e07fee7edbd736f0121100c5c9dfb00ce7d7a699623R387 to sort the files to ensure a consistent ordering.

Reply:
Cool, since both PRs touched this test, I've rebased #330.